src/components/mapPageComponents/mapView/index.tsx

```diff
@@ -75,24 +76,19 @@ const MapView: React.FC<MapViewProps> = ({ blocks, neighborhoods }) => {
   const mapId = '76c08a2450c223d9';
   const markersArray: google.maps.Marker[] = [];
 
-  interface KeyValuePair {
-    key: string;
-    value: string;
-  }
-  // This KeyValuePair array that stores names to be shortHand-ed.
-  // To add a new area, wrap a key value pair with {}.
-  /*
-  const shortHandNames: KeyValuePair[] = [
-    { key: 'North End', value: 'NE' },
-    { key: 'West End', value: 'WE' },
-    { key: 'Leather District', value: 'LD' },
-    { key: 'Beacon Hill', value: 'BH' },
-    { key: 'Back Bay', value: 'BB' },
-    { key: 'Downtown', value: 'DT' },
-    { key: 'Chinatown', value: 'CT' },
-    { key: 'Bay Village', value: 'BV' },
-  ];
-  */
+  // This dictionary that stores names to be shortHand-ed.
+  // To add a new area, add the full name and then the shorthand name seperated by a colon.
+  const shortHandNames: { [fullName: string]: string } = {
+    'North End': 'NE',
+    'West End': 'WE',
+    'Leather District': 'LD',
+    'Beacon Hill': 'BH',
+    'Back Bay': 'BB',
+    Downtown: 'DT',
+    Chinatown: 'CT',
```

Comment:
❓ Is there a reason these aren't in quotes (`'Downtown'` and `'Chinatown'`)?

Reply:
Given the string return type, it seems like it automatically removes it every time. Only strings with more than one word are put in the apostrophe